VmAgent.Core.UnitTests/ExceptionAssert.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Microsoft.Azure.Gaming.VmAgent.Core.UnitTests
```

Comment:
This code is from VmAgent. Once I get confirmed that unit testes I implemented needs this code, I will move this file from VmAgent to VmAgent core so they can share this file.